inputs/radiation/athinput.rad_linearwave_amr

```diff
@@ -0,0 +1,89 @@
+<comment>
+problem   = Radiation beam test
+reference = 
+configure = --prob=radwave
+
+<job>
+problem_id = radwave        # problem ID: basename of output filenames
+
+<output1>
+file_type  = hst       # History data dump
+dt         = 0.01      # time increment between outputs
+
+
+#<output2>
+#file_type  = vtk       # Binary data dump
+#variable   = prim      # variables to be output
+#dt         = 0.01       # time increment between outputs
+
+#<output3>
+#file_type  = rst       # Binary data dump
+#dt         = 0.01       # time increment between outputs
+
+#<output4>
+#file_type  = hdf5       # Binary data dump
+#dt         = 0.01       # time increment between outputs
+#variable   = prim
+
+#<output5>
```

Comment:
@yanfeij FYI there are a bunch of comments/simple fixes from my earlier review still outstanding. Just wanted to make sure you saw them (GitHub hides them since the PR conversation is getting so long)